src/bootman/kernel.c

```diff
@@ -27,6 +27,30 @@
 
 #include "config.h"
 
+static inline const char *strip_slashes(const char *path) {
```

Comment:
This only strips the first instance of a slash, so the name is a bit misleading.

Reply:
Good catch! Renamed it to `strip_leading_slashes` (because paths can have multiple leading slashes for some reason).